README.md

```diff
@@ -37,7 +37,7 @@ Questions can be asked by raising an `Issue`.
 ```
   touch text.txt
   echo "Hallo world" > text.txt
-  git add text.txt
+  git add .
```

Comment:
Using the `.` parameter does work but is more risky. All the time I see people commit many files by mistake, it is safer and recommended to be specific